src/filtrering/filtrering-filter/filterform/hendelser-filterform.tsx

```diff
@@ -3,30 +3,42 @@ import * as React from 'react';
 import {useEffect, useState} from 'react';
 import NullstillValgKnapp from '../../../components/nullstill-valg-knapp/nullstill-valg-knapp';
 import {FiltervalgModell} from '../../../model-interfaces';
-import {hendelserLabels} from '../../filter-konstanter';
+import {hendelserLabels, ulesteEndringer} from '../../filter-konstanter';
 import './filterform.less';
 import {kebabCase} from '../../../utils/utils';
 import {useFeatureSelector} from '../../../hooks/redux/use-feature-selector';
-import {HENDELSE_MEDISINSKBEHANDLING} from '../../../konstanter';
+import {HENDELSE_MEDISINSKBEHANDLING, ULESTE_ENDRINGER} from '../../../konstanter';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import {PopoverOrientering} from 'nav-frontend-popover';
+import {OversiktType} from '../../../ducks/ui/listevisning';
 
 interface HendelserFilterformProps {
     form: string;
     endreFiltervalg: (form: string, filterVerdi: string[]) => void;
     filtervalg: FiltervalgModell;
+    oversiktType: OversiktType;
 }
 
-export function HendelserFilterform({form, filtervalg, endreFiltervalg}: HendelserFilterformProps) {
+export function HendelserFilterform({form, filtervalg, endreFiltervalg, oversiktType}: HendelserFilterformProps) {
     const erMedisinskBehandlingFeatureTogglePa = useFeatureSelector()(HENDELSE_MEDISINSKBEHANDLING);
     const [hendelserValg, setHendelserValg] = useState<string[]>(filtervalg[form]);
+    const [checkboxValg, setCheckboxValg] = useState<string[]>(filtervalg[form]);
+    const ulestTittel = ulesteEndringer.ULESTE_ENDRINGER;
```

Comment:
Er det ikke mulig å heller importe ULESTE_ENDRINGER og bruke den i stedet for å legge den i en variabel?